fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -192,7 +199,7 @@ public void addLogicalResources(List<IDatabaseObject> group, String prefix) {
         // things sensible.
         Table.Builder builder = Table.builder(schemaName, tableName)
                 .setTenantColumnName(MT_ID)
-                .setVersion(FhirSchemaVersion.V0012.vid()) // V0011: is_deleted and last_updated, V0012: version_id
+                .setVersion(FhirSchemaVersion.V0017.vid()) // V0011: is_deleted and last_updated, V0012: version_id
```

Comment:
update comment to say what changes with V0017?

Reply:
Updated